libr/core/cmd_type.c

```diff
@@ -267,8 +267,21 @@ static void save_parsed_type(RCore *core, const char *parsed) {
 	}
 }
 
+static Sdb *TDB_ = NULL; // HACK
+
 static int stdifstruct(void *user, const char *k, const char *v) {
-	return !strncmp (v, "struct", strlen ("struct") + 1);
+	r_return_val_if_fail (TDB_, false);
+	if (!strncmp (v, "struct", strlen ("struct") + 1)) {
+		return true;
+	}
+	if (!strcmp (v, "typedef")) {
+		const char *typedef_key = sdb_fmt ("typedef.%s", k);
+		const char *type = sdb_const_get (TDB_, typedef_key, NULL);
+		if (type && !strncmp (type, "struct ", 7)) {
```

Comment:
for readability we can use r_str_startswith()

Reply:
done 6c632e94cc0bc88a603a036214ed81aa2b431deb